src/modules/types/science/biology/well_plate/view.js

```diff
@@ -138,7 +138,7 @@ define(['modules/default/defaultview', 'src/util/api', 'src/util/color'], functi
         const order = sortArray(nbSamples + controlItems.length);
         addReplicates(samplesList, labelsList, color, replicates, order, mode);
         this.module.controller.createDataFromEvent('onSample', 'list', samplesList);
-        const cellsList = builtPlate(samplesList, labelsList, replicates, nbRows, nbColumns);
+        const cellsList = builtPlate(samplesList, labelsList, nbRows, nbColumns);
```

Comment:
When having so many arguments it is better to use options with default values
like nbRows=8, nbColums=12, replicates=1, etc